src/ReduxStore/selectors/matchedCitiesSelectors.js

```diff
@@ -0,0 +1,13 @@
+import { createSelector } from 'reselect';
+
+export const getMatchedCitiesData = state => state.matchedCities.cities;
+
+export const transformCitiesData = createSelector(getMatchedCitiesData, cities => {
```

Comment:
The only thing here I'd suggest to prefix selectors (made with createSelector utility) with some unified thing, like: selectSomething, or deriveSomething